readme.md

```diff
@@ -191,6 +240,18 @@ Execute a file synchronously.
 
 Returns or throws a [`childProcessResult`](#childProcessResult).
 
+### \$(options?)\`command\`
+
+Same as `execa()` except both file and arguments are specified in a single tagged template string. For example, ``$`echo unicorns` ``is the same as `execa('echo', ['unicorns'])`.
+
+The [`shell` option](#shell) must be used if the `command` uses shell-specific features (for example, `&&` or `||`), as opposed to being a simple `file` followed by its `arguments`.
+
+### $(options?).sync\`command\`
+
+Same as [$(options?)\`command\`](#optionscommand) but synchronous.
+
+Returns or throws a [`childProcessResult`](#childProcessResult).
```

Comment:
The async method should include a similar line IMHO. This is implied by "Same as `execa()`". However, it might be good to be explicit about the return value. What do you think?

Reply:
Addressed here: https://github.com/sindresorhus/execa/pull/510#discussion_r1111455947